homeassistant/components/panasonic_viera/media_player.py

```diff
@@ -83,8 +85,10 @@ def device_info(self):
         return {
             "name": self._name,
             "identifiers": {(DOMAIN, self._device_info[ATTR_UDN])},
```

Comment:
Won't this error if `self._device_info` is None?

Reply:
there's a check that returns `None` right above, haha

Reply:
Good! Thanks!